src/callinput.c

```diff
@@ -793,7 +803,7 @@ int callinput(void) {
 
 	    // Underscore, confirm last exchange.
 	    case '_': {
-		prev_qso();
+		send_standard_message_prev_qso(2);
```

Comment:
In S&P mode SP_TU_MSG shall be sent.

In order to better see the difference I used this config
```
F3=@ [ #
S&P_TU_MSG=[ # XX
```
Working A1BB and the re-sending the exchange:
![image](https://user-images.githubusercontent.com/15141948/201513441-15bc8042-9838-4ada-8c93-ef21866da92f.png)